src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -1875,6 +1875,21 @@
     /**
      * The {@code super} keyword.
      *
+     * <p>For example:</p>
+     * <pre>
+     * super.toString()
+     * </pre>
+     * <p>parses as:</p>
+     * <pre>
+     * |--EXPR -> EXPR
```

Comment:
Please replace `->` with `-&gt;` in all lines and please use updated AST which is generated by latest version of checkstyle.